h2o-r/h2o-package/R/automl.R

```diff
@@ -210,3 +202,19 @@ predict.H2OAutoML <- function(object, newdata, ...) {
   h2o.getFrame(dest_key)
 }
 
+#' Ask the H2O server whether AutoML extension is available
+#' Returns True if a AutoML is available, False otherwise
+#' @export
+h2o.automl.available <- function() {
+  "AutoML" %in% h2o.list_api_extensions()
+}
+
+.h2o.automl.na.error <- function() {
+  message("
+         ******************************************************************************************\n
+         * Please verify that you have the AutoML extension!                                      *\n
+         * Make sure the h2o-automl.jar is stored in h2o-ext directory (relative to your h2o.jar) *\n
```

Comment:
Good point, I think it should ideally point to documentation for getting AutoML and starting on different platforms.